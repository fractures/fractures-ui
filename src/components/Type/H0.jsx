import PropTypes from "prop-types"
import styled from "styled-components"

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H0 = styled.h1`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 8rem;
	line-height: 10rem;
`

H0.propTypes = propTypes

export default H0
