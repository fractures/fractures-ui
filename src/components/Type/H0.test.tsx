import H0 from './H0'

describe('H0', () => {
	const tree = renderer.create(<H0>Extra header</H0>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
