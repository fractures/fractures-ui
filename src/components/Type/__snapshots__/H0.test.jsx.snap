// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H0 Matches snapshot 1`] = `
<h1
  className="sc-bdVaJa dqxxAd"
>
  Extra header
</h1>
`;
